autogoal/contrib/ensemble/_stacking.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+from sklearn import svm
+from sklearn.naive_bayes import GaussianNB
+
+
+class StackingEnsemble:
+
+    def __init__(self, X, y):
```

Comment:
Instead of receiving the `X` and `y` in the constructor, we prefer to have a `fit` method like `sklearn`. The constructor should only receive the parameters necessary to define the ensemble, i.e., number of layers, types of the internal algorithms, etc.

Reply:
I see, the method that I called `forward_pass()` does the fit, then I just should change it and make it so that it takes the input data ?

Reply:
You could rename it, or leave it as an internal method `_forward_pass` and then just have a `fit` that calls it and validates, dunno, that `X` and `y` are not `None`, have the same length, etc.

What I do believe is necessary is to make `X` and `y` parameters of that method instead of the constructor, to be compatible with all the other APIs.